utils/composer-packages/src/Rector/RemovePackageVersionsRector.php

```diff
@@ -41,29 +41,27 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        if (! property_exists($node, 'stmts')) {
-            return null;
-        }
+        /** @var Closure $closure */
+        $closure = $node;
 
```

Comment:
Thanks to validation method above, we know that the `$node` is always `Closure` node. Tools cannot handle method calls type check, so we have to remind them the type.